.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tests/spectestcase"]
+	path = tests/spectestcase
+	url = https://github.com/ontio/testsuite
```

Comment:
Our forked testsuite disabled all testcases related to floating-point operation and I do not have enough time to check that part. We can replace it with `https://github.com/WebAssembly/testsuite` and fix the floating-point operation in other PR.

Reply:
it seems we've grown too many "testsuite" datasets over time...

perhaps not for this PR, but it would be great to streamline a bit all of those...